galaxy.yml

```diff
@@ -16,7 +16,7 @@ dependencies:
   community.mysql: 3.6.0
   kubernetes.core: 2.4.0
   openstack.cloud: 1.7.0
-  vexxhost.ceph: 3.0.1
+  vexxhost.ceph: 3.0.2
```

Comment:
Let's move towards `>=3.0.2` instead, we want to move away from pinned since it is causing issues when trying to update things.